src/Libraries/PythonNodeModelsWpf/PythonNode.cs

```diff
@@ -34,6 +34,29 @@ public void CustomizeView(PythonNode nodeModel, NodeView nodeView)
             var editWindowItem = new MenuItem { Header = PythonNodeModels.Properties.Resources.EditHeader, IsCheckable = false };
             nodeView.MainContextMenu.Items.Add(editWindowItem);
             editWindowItem.Click += delegate { EditScriptContent(); };
+            // If it is a Debug build, display a python engine switcher
+            if (Dynamo.Configuration.DebugModes.IsEnabled("Python3DebugMode"))
+            {
+                var pythonEngineVersionMenu = new MenuItem { Header = PythonNodeModels.Properties.Resources.PythonNodeContextMenuEngineSwitcher, IsCheckable = false };
+                nodeView.MainContextMenu.Items.Add(pythonEngineVersionMenu);
+                pythonEngine2Item.Click += delegate { UpdateToPython2Engine(); };
```

Comment:
hmm - worth unsubscribing these on dispose/unload?

Reply:
@mjkkirschner Thanks, I was referring to the legacy code and did not pay attention. Addressed